extensions/azure/cosmos/contract-negotiation-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/negotiation/store/CosmosContractNegotiationStoreIntegrationTest.java

```diff
@@ -316,35 +374,25 @@ void findAll_noQuerySpec() {
     @Test
     void findAll_verifyPaging() {
 
-        var all = IntStream.range(0, 10).mapToObj(i -> generateDocument())
-                .peek(d -> container.createItem(d))
-                .map(ContractNegotiationDocument::getId)
-                .collect(Collectors.toList());
+        var all = IntStream.range(0, 10).mapToObj(i -> generateDocument()).peek(d -> container.createItem(d)).map(ContractNegotiationDocument::getId).collect(Collectors.toList());
 
         // page size fits
-        assertThat(store.queryNegotiations(QuerySpec.Builder.newInstance().offset(3).limit(4).build())).hasSize(4)
-                .extracting(ContractNegotiation::getId).isSubsetOf(all);
+        assertThat(store.queryNegotiations(QuerySpec.Builder.newInstance().offset(3).limit(4).build())).hasSize(4).extracting(ContractNegotiation::getId).isSubsetOf(all);
 
     }
 
     @Test
     void findAll_verifyPaging_pageSizeLargerThanCollection() {
 
-        var all = IntStream.range(0, 10).mapToObj(i -> generateDocument())
-                .peek(d -> container.createItem(d))
-                .map(ContractNegotiationDocument::getId)
-                .collect(Collectors.toList());
+        var all = IntStream.range(0, 10).mapToObj(i -> generateDocument()).peek(d -> container.createItem(d)).map(ContractNegotiationDocument::getId).collect(Collectors.toList());
```

Comment:
nit: in all these tests, it seemed more legible to split chain on multiple lines

Reply:
I tend to agree for >3 chained statements.